src/cinder/PolyLine.cpp

```diff
@@ -157,8 +168,6 @@ polygon convertPolyLinesToBoostGeometry( const std::vector<PolyLine<T> > &a )
 template<typename T>
 std::vector<PolyLine<T> > PolyLine<T>::calcUnion( const std::vector<PolyLine<T> > &a, std::vector<PolyLine<T> > &b )
 {
-	typedef boost::geometry::model::polygon<boost::geometry::model::d2::point_xy<double> > polygon;
```

Comment:
These can just use the typedef in the anonymous namespace where the conversion functions are defined.